core/activity/handler.go

```diff
@@ -30,14 +30,16 @@ import (
 type Handler interface {
 	OnMessage(context *TurnContext) (schema.Activity, error)
 	OnInvoke(context *TurnContext) (schema.Activity, error)
+	OnConversationUpdate(context *TurnContext) (schema.Activity, error)
 }
 
 // HandlerFuncs is an adaptor to let client program specify as many or
 // as few functions to handle events of the connector service while still implementing
 // Handler.
 type HandlerFuncs struct {
-	OnMessageFunc func(turn *TurnContext) (schema.Activity, error)
-	OnInvokeFunc  func(turn *TurnContext) (schema.Activity, error)
+	OnMessageFunc            func(turn *TurnContext) (schema.Activity, error)
+	OnInvokeFunc             func(turn *TurnContext) (schema.Activity, error)
+	OnConversationUpdateFunc func(context *TurnContext) (schema.Activity, error)
```

Comment:
```suggestion
	OnConversationUpdateFunc func(turn *TurnContext) (schema.Activity, error)
```
minor: let's follow the uniform pattern